### DISCUSSION TITLE
QRT not working with rerelease

### DISCUSSION THREAD

**Discussion by SonicBoomin** [sultim-t/vkquake-rt#42] [Q&A]
I followed all the details with the rereleased version on installation.

- Quake loads fine.
- Incorrect version loading up possibly.
- Does anyone have a solution to this error?

**Comment by ChemGuy1611** [comment]
What error? Make sure you use v1.0.2 if you are on AMD.  I used the Xbox Game Pass version of the game (it is the 2021 ReRelease) and it worked fine.

**Comment by andreottigiulio** [comment]
> I followed all the details with the rereleased version on installation.
> 
> * Quake loads fine.
> * Incorrect version loading up possibly.
> * Does anyone have a solution to this error?

Same here,
playing quake with RT it AWESOME. i finished it at Nightmare skill in a couple of days. 
i manage also to boot with this version:
Scourge of Armagon
Dissolution of Eternity
Dimension of the Past
All flawlessly with RT support.

But when it comes to "Dimension of the Machine" i got this error on starting up new campaign
![image](https://github.com/user-attachments/assets/9ae6d03b-1f91-4b55-a8ae-2e2feefe0385)

im still able to startup certain maps though (mge1m1 and hub for example) but there are some glitches and the same error will appear on hub map if i advance forward...
I downloaded and compiled latest vkquake sources and there is no problem with that but adding RT support is way beyond my skills.
i am hoping for someone's help.
Much love for your work!
